platform/pc/service.c

```diff
@@ -374,7 +380,14 @@ void kernel_shutdown(int status)
 {
     shutting_down = true;
     if (root_fs) {
-        storage_sync(closure(heap_general(&heaps), sync_complete, status));
+        heap h = heap_locked(&heaps);
+        status_handler completion = closure(h, sync_complete, status);
+        merge m = allocate_merge(h, completion);
+        status_handler sh = apply_merge(m);
+        if ((status != 0) && table_find(filesystem_getroot(root_fs), sym(crash_report)))
```

Comment:
do we need an extra bool for this? wouldn't having the RADAR_API env token specified should be enough?

Reply:
OK, I'll get rid of the crash_report flag, and use just the RADAR_KEY environment variable